src/app-layout/__tests__/global-breadcrumbs.test.tsx

```diff
@@ -135,30 +135,31 @@ describeEachAppLayout({ themes: ['refresh-toolbar'], sizes: ['desktop'] }, () =>
     expect(findRootBreadcrumb().getElement()).toHaveTextContent('Second');
   });
 
-  test('when multiple app layouts rendered, only the last instance receives breadcrumbs', async () => {
+  test('when multiple app layouts rendered, only the first instance receives breadcrumbs', async () => {
     await renderAsync(
       <>
         <AppLayout {...defaultAppLayoutProps} data-testid="first" />
         <AppLayout
           {...defaultAppLayoutProps}
           data-testid="second"
+          navigationHide={true}
```

Comment:
Prevent warnings about duplicate navigations used between app layouts. Not relevant for the test about breadcrumbs, but still annoying to see in the console